test/test_unit.jl

```diff
@@ -610,6 +610,22 @@ Cassette.@context Ctx
       end
     end
   end
+
+  @timed_testset "EulerAcoustics" begin
```

Comment:
This test is way too expensive. If I am not mistaken, it takes roughly 10 minutes:
https://github.com/trixi-framework/Trixi.jl/pull/712/checks?check_run_id=3507070514#step:7:1093

Thus, if there is no way to just create the callbacks without running the actual test, I would argue we should skip the tests. But shouldn't it be possible to create the callbacks without actually running the simulation?

Reply:
Can you please check once more to be sure - maybe manually on your machine as well? I did a quick search in the CI logs, and it seems like the testset still takes more than 4 minutes to run (263 seconds).
https://github.com/trixi-framework/Trixi.jl/pull/712/checks?check_run_id=3543760971#step:7:836

Reply:
That's still quite long. I checked it on my machine and it takes about 39 seconds with `coverage=true` and 9 seconds without it. Unfortunately, I don't see any way to further simplify this test.

Reply:
@ranocha What's your take on this? 4 minutes seem to be excessive for just testing `show`, but it is the weird `misc_part1` job that sometimes is annoyingly slow. I guess it would be OK to just leave `show` untested in this case.

Reply:
Yes, remove the test

Reply:
Removed the test in 6124dac.